Docs/user_guide/modules/markups.md

```diff
@@ -163,6 +237,24 @@ ROI settings section:
 - Display ROI: show/hide the ROI in all views.
 - Interactive mode: allow changing the ROI position, orientation, and size in views using interaction handles. If interaction handles are disabled, the ROI may still be changed by moving control points (unless control points are locked, too).
 
+## Creating Template Landmarks
+
+In order to define a workflow in which a known list of markups must be placed, it is possible to save/load predefined markups lists to use as templates.
+
+Defining a template:
+  1. Create a new point list using the toolbar, or the create button in the Markups module. You may want to exit place mode by clicking on the toolbar, or by right-clicking in a view.
+  1. In the Markups module, select the newly created points list
+  1. In the Control Points - Advanced section, create the required number of control points for the template by clicking on the Add Control Point button.
+  1. Set the name/description of the control points in the control points table.
+  1. To prevent points from being added or removed, click on the "Fixed list of points" button at the top of the control points section.
+  1. Export the template to file by right-clicking on the points list in the markups module, or in the subject hierarchy, and select "Export to file...".
```

Comment:
```suggestion
  1. Create a new point list using the toolbar, or the create button in the Markups module. You may want to exit place mode by clicking on the toolbar, or by right-clicking in a view.
  2. In the Markups module, select the newly created points list
  3. In the Control Points - Advanced section, create the required number of control points for the template by clicking on the Add Control Point button.
  4. Set the name/description of the control points in the control points table.
  5. To prevent points from being added or removed, click on the "Fixed list of points" button at the top of the control points section.
  6. Export the template to file by right-clicking on the points list in the markups module, or in the subject hierarchy, and select "Export to file...".
```